GeosCore/drydep_mod.F90

```diff
@@ -1591,730 +1594,764 @@ SUBROUTINE DEPVEL( Input_Opt, State_Chm, State_Diag, State_Grid, &
           !*    Loop over species; species-dependent corrections to resistances
           !*    are from equations (6)-(9) of Wesely [1989].
           !*
-          DO 160  K = 1,NUMDEP
+          DO K = 1,NUMDEP
 
              ! Save F0(K) in a private variable to avoid diffs due
              ! to parallelization -- Bob Yantosca (17 May 2023)
              F0_K = F0(K)
 
-             !** exit for non-depositing species or aerosols.
-             IF (.NOT. LDEP(K) .OR. AIROSOL(K)) GOTO 155
+             !** Only consider dry depositing gas-phase species:
+             !IF (.NOT. LDEP(K) .OR. AEROSOL(K)) GOTO 155  ! If LDEP and not an aerosol proceed:
+             IF ( LDEP(K) .AND. ( .NOT. AEROSOL(K) ) ) THEN
 
-             ! Test for special treatment for O3 drydep to ocean
-             N_SPC = State_Chm%Map_DryDep(K)
-             IF ((N_SPC .EQ. ID_O3) .AND. (II .EQ. 11)) THEN
+                IF ( .NOT. LDEP(K) ) THEN
+                   PRINT*, 'processing non-depositing species. FIX!'
+                   STOP
+                ENDIF
```

Comment:
Instead of STOP, we should return the error code to the calling program for graceful termination. 
```F90
IF (  .not. LDEP(K) ) THEN
   errMsg = 'Processing a species that does not dry-deposit.'
   CALL GC_Error( errMsg, RC, thisLoc )
   RETURN
ENDIF
```

Reply:
Actually this IF block will never execute so we can remove it.